client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
If we're going to use a hash for the unique ID, there isn't really a point to also having the current "case_id". Ordering information is given by the unlocking timestamps.

It's possible (but unlikely) that two cases in the same test have the same contents, e.g.

```
>>> x = 5
>>> square(x)
25
>>> square(-x)
25
```
and
```
>>> x = -5
>>> square(x)
25
>>> square(-x)
25
```

In this case, the second prompt in both cases would record
```
{'question': '>>> square(-x)',
 'answer': '25'}
```